### PR TITLE
AMB-863 Fix try this API feature

### PR DESCRIPTION
**Pull request by RyanThomas1214**
## Summary
* Routine Change
* :exclamation: Breaking Change
* :robot: Operational or Infrastructure Change
* :sparkles: New Feature
* :warning: Potential issues that might be caused by this change

Add any other relevant notes or explanations here. **Remove this line if you have nothing to add.**

## Reviews Required
* [x] Dev
* [ ] Test
* [ ] Tech Author
* [ ] Product Owner

## Review Checklist
:information_source: This section is to be filled in by the **reviewer**.

* [ ] I have reviewed the changes in this PR and they fill all or part of the acceptance criteria of the ticket, and the code is in a mergeable state.
* [ ] If there were infrastructure, operational, or build changes, I have made sure there is sufficient evidence that the changes will work.
* [ ] I have ensured the changelog has been updated by the submitter, if necessary.

### CODE REVIEW

File: specification/booking-and-referral.yaml [hunk 1]
```diff
@@ -1,6 +1,63 @@
...
+    
+    The API is synchronous.
+    
+    The following describes the end-to-end process:
+    1. The sending system sends the payload to the Booking and Referrals API, along with an identifier header.
+    2. The Booking and Referrals API directs the payload to the target backend associated with the identifier header.
+    3. The sending system recieves the response.
+    
+    ## Who can use this API
+    This API can only be used where there is a legal basis to do so. Make sure you have a valid use case before you go too far with your development. You must do this before you can go live (see ‘Onboarding’ below).
```
Comment (SachinPath) [inline specification/booking-and-referral.yaml:15]: Has onboarding started for this API? I cannot see the "Onboarding" section in the overview section?  
If not then remove the (see ‘Onboarding’ below).

File: specification/booking-and-referral.yaml [hunk 2]
```diff
@@ -1,6 +1,63 @@
...
+    This API is in [alpha](https://digital.nhs.uk/developer/guides-and-documentation/reference-guide#api-status), meaning:
+      * it is available for testing in integration environment
+      * we expect to make breaking changes based on developer feedback
+    
+    ### Roadmap
+    To see our roadmap, or to suggest, comment or vote on features for this API, see our [interactive product backlog](https://nhs-digital-api-management.featureupvote.com/?order=popular&filter=allexceptdone&tag=fhir-converter&deleted=0#controls).
+    If you have any other queries, please [contact us](https://digital.nhs.uk/developer/help-and-support).
+    
+    ## Technology
+    This API uses HTTP POST to submit data.
```
Comment (SachinPath) [inline specification/booking-and-referral.yaml:27]: See my comment on "This API is synchronous."

See [PDS FHIR API - Technology](https://digital.nhs.uk/developer/api-catalogue/personal-demographics-service-fhir#api-description__technology) section.

File: specification/booking-and-referral.yaml [hunk 3]
```diff
@@ -1,6 +1,63 @@
...
+    Specifically, it is aligned with [FHIR UK Core](https://digital.nhs.uk/services/fhir-uk-core), which is built on FHIR Release 4, or [FHIR GP Connect](https://digital.nhs.uk/services/gp-connect), which is built on FHIR STU3.
+    
+    ## Network access
+    This API is available on the internet.
+    
+    For more details, see [Network access for APIs](https://digital.nhs.uk/developer/guides-and-documentation/network-access-for-apis).
+    
+    ## Security and authorisation
+    
+    ### Application-restricted access
```
Comment (SachinPath) [inline specification/booking-and-referral.yaml:39]: Do we have more than one access mode for this API?

If not, remove the header for "Application-restricted access"

File: specification/booking-and-referral.yaml [hunk 4]
```diff
@@ -1,6 +1,63 @@
...
+    This access mode is [application-restricted](https://digital.nhs.uk/developer/guides-and-documentation/security-and-authorisation#application-restricted-apis),
```
Comment (SachinPath) [inline specification/booking-and-referral.yaml:40]: If there is only one access mode, change to "This API is [application-restricted](https://digital.nhs.uk/developer/guides-and-documentation/security-and-authorisation#application-restricted-apis),......."

File: specification/booking-and-referral.yaml [hunk 5]
```diff
@@ -1,6 +1,63 @@
...
+    ### Environments and testing
+    | Environment       | Base URL                                                               |
+    | ----------------- | ---------------------------------------------------------------------- |
+    | Integration test  | `https://int.api.service.nhs.uk/fhir-converter`                        |
+    | Production        | Not yet available                                                      |
+    
+    Our [integration test environment](https://digital.nhs.uk/developer/guides-and-documentation/testing#integration-testing):
+    - is for formal integration testing
+    - is for receiving data quality reports of test data
+
```
Comment (SachinPath) [inline specification/booking-and-referral.yaml:58]: Does this API provide any data quality reports? 

I think the bullet point is applicable to the "[Ambulance data submission API.](https://digital.nhs.uk/developer/api-catalogue/ambulance-data-submission-fhir#api-description__security-and-authorisation)" 

Mention relevant benefits for the Integration environment of BaRS API.

**Review by SachinPath — CHANGES_REQUESTED** [review]

**Review by Valentinolucas — APPROVED** [review]

**Pull request merged**